### ISSUE TITLE
wPaint in Android

### ISSUE THREAD

**Issue by luiscruz** [websanova/wPaint#3]
Hi,

Thanks for the plugin, it is great!
However, I tried it with an Android tablet and it does't work properly. One of the advantages of not using a Flash plugin is to be able to use it on mobile applications.

Hopefully, it can be a small fix and soon we will be able to use it on Android.

**Comment by websanova** [comment]
Do you have any errors output, screenshots, anything of use for me to work
with would be great,

Thanks,

Rob

**Comment by luiscruz** [comment]
I took a screenshot on Android 4.0.4, in which I tried to draw 'L' with the pencil:
http://cl.ly/image/2b3A0t0f2n3g

I took also a screenshot on an iOS 6.0.1, trying to write an 'L', and it worked as expected. However, the color picker did't work:
http://cl.ly/image/3O2N0q270E3c

After a refresh, the color picker worked. And I think the same behavior happened when I first tried with the Android device.
